site/content/docs/11-deploying.md

```diff
@@ -38,15 +35,16 @@ const timestamp = process.env.SAPPER_TIMESTAMP; // instead of `import { timestam
 const ASSETS = `cache${timestamp}`;
 
 export default {
-	/* ... */
-	plugins: [
-		/* ... */
-		replace({
-			/* ... */
-			'process.env.SAPPER_TIMESTAMP': process.env.SAPPER_TIMESTAMP || Date.now()
-		})
-	]
-}
+  /* ... */
+  plugins: [
+    /* ... */
+    replace({
+      /* ... */
+      'process.env.SAPPER_TIMESTAMP':
```

Comment:
I think the docs mostly use tabs for code indentation, so should probably leave that for consistency. Also, I think this was easier to read before when this line and the next line were on one line